scripts/container.sh

```diff
@@ -16,7 +16,7 @@ then
 elif [ "$(command -v docker)" ]
 then
 	docker build -t "${TAG}" contrib/docker
-	docker run -it --rm --volume="$(pwd):/gluon" "${TAG}"
+	docker run -it --rm --user $(id -u):$(id -g) --volume="$(pwd):/gluon" "${TAG}"
```

Comment:
```suggestion
	docker run -it --rm --user "$(id -u)":"$(id -g)" --volume="$(pwd):/gluon" "${TAG}"
```

At least shellcheck throws a warning in this part. (_Quote this to prevent word splitting_)